optimum/onnxruntime/trainer.py

```diff
@@ -1500,6 +1514,7 @@ def _export(
                 output_dir=model_path,
                 output_names=output_names,
                 device=device,
+                disable_dynamic_axes_fix=True,  # onnxruntime floating point exception (core dumped)
```

Comment:
Do you know what there is this exception? Can't it be avoided?

Reply:
It happens when export a ORT trained model with task `causal-lm-with-past` and only for the with-past decoder.
It tried automatically to fix the dynamic axes name of `logits dim=1`, and error occurs in this line:
https://github.com/huggingface/optimum/blob/4100c384127995de772a7a5b4a22df3b27e27e73/optimum/exporters/onnx/base.py#L335
I doubted it was the past-key-values as it only appeared for with past decoder and I checked,  but the dummy pkv look correct to me. Search a little bit, seems that this type of error could appear when reload model under different device / opset etc... I wonder if it is due to the fact that ORT trained model will have some Microsoft domain op while exporting to ONNX and `SessionOptions` need to precisely configured, but I did not dig further as it is special case and not impacting the ORTTrainer anyway.